test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -282,6 +288,34 @@ def test_emacs(image):
     _run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+@pytest.mark.integration("sagemaker python sdk")
+def test_sm_pysdk_2(tensorflow_training):
```

Comment:
Please use `image` as input here, and then skip inference images.

Also, rather than running this test only on TensorFlow images, could you `xfail` on MXNet and PyTorch images so that it can be addressed in the subsequent PR meant for MX and PT dockerfiles?

Reply:
Please see the "param" description - this will be changed to "training" subsequently. "image" is not the appropriate fixture here.